.github/workflows/node.js.yml

```diff
@@ -5,26 +5,45 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - main
+      - stable*
   pull_request:
-    branches: [ main ]
+    branches:
+      - main
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
+    name: node
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
 
-    strategy:
-      matrix:
-        node-version: [12.x, 14.x, 15.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+      - name: Read package.json node and npm engines version
+        uses: skjnldsv/read-package-engines-version-actions@v1.1
+        id: versions
+        with:
+          fallbackNode: '^12'
+          fallbackNpm: '^6'
```

Comment:
Not what you want, I said "imspire yourself" not copy fully :wink: 
We use that for our old legacy branches. You want at least node12 and npm 7 I assume

Reply:
> Not what you want, I said "imspire yourself" not copy fully :wink: 

Oooh yes of course ! I wanted test it before the custom ^^